scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -269,3 +199,76 @@
     }
   }
 }
+
+
+//== Nav tabs in a panel-default
+.panel-default {
+
+  // Horizontal nav tabs
+  .nav-tabs:not(.nav-tabs-stacked) {
+    $padding-x: nth($crm-panel-subheading-padding, 2);
```

Comment:
Thought about that, but in this case i think a one-liner would be harder to understand, thus i left thing a little more explicit